cmd/resource-topology-exporter/main.go

```diff
@@ -16,22 +30,49 @@ import (
 	"github.com/k8stopologyawareschedwg/resource-topology-exporter/pkg/podrescli"
 	"github.com/k8stopologyawareschedwg/resource-topology-exporter/pkg/resourcemonitor"
 	"github.com/k8stopologyawareschedwg/resource-topology-exporter/pkg/resourcetopologyexporter"
+
+	"github.com/openshift-kni/resource-topology-exporter/pkg/podrescompat"
```

Comment:
Why we're having those packages under openshift-kni and not the standard working group?

Reply:
The main reason from the wg perspective this feature is not really interesting: we are addressing a corner case the majority of users don't care about.